docs/src/main/paradox/scala/http/common/http-model.md

```diff
@@ -333,3 +333,27 @@ need to register the custom media type in the server's settings by configuring `
 
 You may also want to read about MediaType [Registration trees](https://en.wikipedia.org/wiki/Media_type#Registration_trees), in order to register your vendor specific media types
 in the right style / place.
+
+<a id="registeringcustomstatuscodes"></a>
+## Registering Custom Status Codes
+
+Similarily to media types, Akka HTTP [predefines](https://github.com/akka/akka-http/blob/master/akka-http-core/src/main/scala/akka/http/scaladsl/model/StatusCode.scala#L102)
```

Comment:
This link is volatile, I suggest linking to the scaladoc with `@scaladoc:[predefines](akka.http.scaladsl.model.StatusCodes$)`